public/app/features/alerting/unified/components/receivers/form/ReceiverForm.tsx

```diff
@@ -159,7 +159,7 @@ export function ReceiverForm<R extends ChannelValues>({
               variant="secondary"
               onClick={() => append({ ...defaultItem, __id: String(Math.random()) } as R)}
             >
-              New contact point type
+              New contact point receiver
```

Comment:
In here we should use integration as well as above, right?

Reply:
This I think should be kept as receiver. You create one or more receivers, and each receiver can contain different or similar integrations (i.e. slack, email, etc), or does receiver refer to a contact point?

Reply:
Looking at the Prometheus docs I think Sonia is right!